addons-l10n/en/rename-broadcasts.json

```diff
@@ -1,5 +1,5 @@
 {
-  "rename-broadcasts/RENAME_BROADCAST": "Rename broadcast",
-  "rename-broadcasts/RENAME_BROADCAST_TITLE": "Rename all \"{name}\" broadcasts to:",
-  "rename-broadcasts/RENAME_BROADCAST_MODAL_TITLE": "Rename Broadcast"
+  "rename-broadcasts/RENAME_BROADCAST": "Rename message",
+  "rename-broadcasts/RENAME_BROADCAST_TITLE": "Rename all \"{name}\" messages to:",
+  "rename-broadcasts/RENAME_BROADCAST_MODAL_TITLE": "Rename Message"
```

Comment:
Please revert these.

Reply:
Removed for this PR, will worry about them again later.